### PR TITLE
Allow manually installing `strong-oracle`.

### PR DESCRIPTION
**Pull request by ikogan**
Helps with https://github.com/strongloop/loopback-connector-oracle/issues/53.

### CODE REVIEW

**Comment by slnode** [comment]
Can one of the admins verify this patch? To accept patch and trigger a build add comment ".*ok\W+to\W+test.*"

File: pkginstall.js [hunk 1]
```diff
@@ -27,35 +27,38 @@ if (!dest) {
...
-download(url, version, dest, function (err, result) {
-  if (err) {
-    process.exit(1);
-  }
-
-  // Now try to run post-installation scripts
-  var inst_dir = path.dirname(process.argv[1]);
+// Check to see if `strong-oracle` is already installed, in case
+// they'd prefer compiling it themselves
+if(!fs.existsSync(path.join(parent, 'strong-oracle'))) {
```
Comment (raymondfeng) [inline pkginstall.js]: We probably need to allow force download if `npm --force` is used.

Comment (ikogan) [inline pkginstall.js]: Should be done I think.

**Comment by dhmlau** [comment]
Closing as abandoned.

**Pull request closed without merging**